go/libkb/api.go

```diff
@@ -204,14 +204,18 @@ func (c *countingReader) numRead() int {
 // Shared code
 //
 
-// The returned response, if non-nil, should have
-// DiscardAndCloseBody() called on it.
-func doRequestShared(api Requester, arg APIArg, req *http.Request, wantJSONRes bool) (_ *http.Response, jw *jsonw.Wrapper, err error) {
+func nullFinisher() {}
+
+// If finisher is returned non-nil, the finisher() should be called after the
```

Comment:
Isn't finisher always non-nil?  And thus `finisher()` should always be called after the response is no longer needed?